mautic/mautic
# Message queue report

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds a new report group for channel messages, and a new message queue report data source.
[//]: # ( As applicable: )
#### Steps to test this PR:
1. Apply PR, clear cache. 
2. Go to reports and create a new report
3. You should now see new report group 'Channel Messages' and a new report data source for 'Queued messages'. Once selected add the columns, [Message channel, Message Id, Priority, Max attempts, Total attempts, Success, Status, Last attempt date, Date sent, Scheduled Date, Date published] to view all related data for the queued messages, select Contact email to make more sense of which the contact the data is related to.

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
